lms/djangoapps/api_manager/users/tests.py

```diff
@@ -1407,6 +1407,15 @@ def test_user_courses_grades_list_get(self):
         self.assertEqual(response.data['current_grade'], 0.73)
         self.assertEqual(response.data['proforma_grade'], 0.9375)
 
+        test_uri = '{}/{}/courses/grades'.format(self.users_base_uri, user_id, unicode(course.id))
```

Comment:
Since unicode(course.id) isn't injected into the test_uri, should it even be here?